tpu_mtj_backend.py

```diff
@@ -1416,3 +1419,6 @@ def callback(model_dict, f, **_):
                 model     = GPTNeoForCausalLM.from_pretrained(vars.model, revision=vars.revision, cache_dir="cache")
 
     #network.state = network.move_xmap(network.state, np.zeros(cores_per_replica))
+    global shard_xmap, batch_xmap
```

Comment:
Was this added by mistake?

```
Traceback (most recent call last):
  File "/content/KoboldAI-Client/aiserver.py", line 10287, in <module>
    load_model(initial_load=True)
  File "/content/KoboldAI-Client/aiserver.py", line 2792, in load_model
    import tpu_mtj_backend
  File "/content/KoboldAI-Client/tpu_mtj_backend.py", line 1422
    global shard_xmap, batch_xmap
    ^
SyntaxError: name 'shard_xmap' is assigned to before global declaration

```

Reply:
I messed up when cherry picking from united. The declaration above with the other lines needed to be moved down with it otherwise you get a error during the tensor conversion.